docs/conventions/branch.md

```diff
@@ -0,0 +1,5 @@
+## Branch
+
+- A branch-name must start with an issue number and issue-name. 
+- Following this a slash (/) with any aditional comments.
+- Example "1/vise-videoer-på-skjerm/first-attempt"
```

Comment:
Noe her

Reply:
Står også på mitt her, dog mer åpen for forandring.